### ISSUE TITLE
Had a problem when i release a production apk using this lib

### ISSUE THREAD

**Issue by Albert0405** [inProgress-team/react-native-youtube#363]
```
    "react": "16.8.3",
    "react-native": "0.59.1",
```

```
* What went wrong:
Execution failed for task ':react-native-youtube:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Andr
oid resource linking failed
  error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
  error: resource android:style/TextAppearance.Material.Widget.Button.Colored not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values-v26\values-v26.xml:7: error: resource android:attr/colorError not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values-v26\values-v26.xml:11: error: resource android:attr/colorError not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values-v26\values-v26.xml:15: error: style attribute 'android:attr/keyboardNavigationClus
ter' not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values-v28\values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values-v28\values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2711: error: resource android:attr/fontStyle not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2712: error: resource android:attr/font not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2713: error: resource android:attr/fontWeight not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2714: error: resource android:attr/fontVariationSettings not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2715: error: resource android:attr/ttcIndex not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2796: error: resource android:attr/startX not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2799: error: resource android:attr/startY not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2802: error: resource android:attr/endX not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2805: error: resource android:attr/endY not found.
  C:\Local D\ML Guide\MLGuide\node_modules\react-native-youtube\android\build\intermediates\res\merg
ed\release\values\values.xml:2813: error: resource android:attr/offset not found.
  error: failed linking references.
```

**Comment by felansu** [comment]
thanks bro, +1

**Comment by felansu** [comment]
i resolve just now changing my app>android>build.gradle to (see the subprojects line):

```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    ext {
        buildToolsVersion = "28.0.3"
        minSdkVersion = 16
        compileSdkVersion = 28
        targetSdkVersion = 28
        supportLibVersion = "28.0.0"
    }
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.3.1'

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
    subprojects { subproject ->
        afterEvaluate {
            if ((subproject.plugins.hasPlugin('android') || subproject.plugins.hasPlugin('android-library'))) {
                android {
                    compileSdkVersion rootProject.ext.compileSdkVersion
                    buildToolsVersion rootProject.ext.buildToolsVersion
                }
            }
        }
    }
}

allprojects {
    repositories {
        mavenLocal()
        google()
        jcenter()
        maven {
            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
            url "$rootDir/../node_modules/react-native/android"
        }
        maven { url "https://jitpack.io" }
        maven { url "https://maven.google.com" }
    }
}
```

**Comment by RickWoltheus** [comment]
> i resolve just now changing my app>android>build.gradle to (see subprojects):
> 
> ```
> // Top-level build file where you can add configuration options common to all sub-projects/modules.
> 
> buildscript {
>     ext {
>         buildToolsVersion = "28.0.3"
>         minSdkVersion = 16
>         compileSdkVersion = 28
>         targetSdkVersion = 28
>         supportLibVersion = "28.0.0"
>     }
>     repositories {
>         google()
>         jcenter()
>     }
>     dependencies {
>         classpath 'com.android.tools.build:gradle:3.3.1'
> 
>         // NOTE: Do not place your application dependencies here; they belong
>         // in the individual module build.gradle files
>     }
>     subprojects { subproject ->
>         afterEvaluate {
>             if ((subproject.plugins.hasPlugin('android') || subproject.plugins.hasPlugin('android-library'))) {
>                 android {
>                     compileSdkVersion rootProject.ext.compileSdkVersion
>                     buildToolsVersion rootProject.ext.buildToolsVersion
>                 }
>             }
>         }
>     }
> }
> 
> allprojects {
>     repositories {
>         mavenLocal()
>         google()
>         jcenter()
>         maven {
>             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
>             url "$rootDir/../node_modules/react-native/android"
>         }
>         maven { url "https://jitpack.io" }
>         maven { url "https://maven.google.com" }
>     }
> }
> ```

also fixes the same issue on rn 0.59.10

**Comment by davidohayon669** [comment]
solved in newer beta
